.circleci/config.yml

```diff
@@ -87,29 +129,48 @@ jobs:
       - run:
           name: Run Client Tests
           command: |
-              npm run test-client:ci $(circleci tests glob "client/**/test/*.js" "client/**/test/*.jsx" | circleci tests split --split-by=timings)
+            JEST_JUNIT_OUTPUT="$CIRCLE_TEST_REPORTS/client/test-results-client.xml" npx --no-install \
+              jest                                    \
+                --ci                                  \
+                --reporters=default                   \
+                --reporters=jest-junit                \
+                --config=test/client/jest.config.json \
+                --maxWorkers=2                        \
```

Comment:
why `parallelism: 6` and `maxWorkers=2`?

Reply:
`parallelism` are the containers that will run this job in parallel. It is defined for the job, so 6 containers will run each step of this job. 6 seemed to work well, but it's something that we should tweak in the future. If job queues tend to form, we might try lowering this a bit. I'd expect that with lower parallelism jobs would take a bit longer, but we'd have more containers free to run more jobs at the same time.

---

`maxWorkers` defines the maximum size of the worker pool Jest uses to run tests in parallel. CircleCI [recommands](https://circleci.com/docs/2.0/collect-test-data/#jest) `--runInBand`, which is analogous to `--maxWorkers=1`, but I believe those docs are outdated.  I did not seem improvements with `runInBand` or issues when using 2 workers.

Me theory for why 2 workers per container is a sweet spot is the [`resource_class`](https://circleci.com/docs/2.0/configuration-reference/#resource_class). We're using the default, so we'll have 2 vCPUs, one for each worker. You can confirm the container resources at the top of a job:

![resources](https://user-images.githubusercontent.com/841763/45802924-72c02700-bcb8-11e8-8636-5e61baabd6d3.png)